lib/canary/plugs.ex

```diff
@@ -193,11 +193,12 @@ defmodule Canary.Plugs do
     current_user = Dict.fetch! conn.assigns, current_user_name
     action = get_action(conn)
     is_persisted = persisted?(opts)
+    non_id_actions = if opts[:non_id], do: ([:index, :new, :create] ++ opts[:non_id]), else: [:index, :new, :create]
```

Comment:
Would you mind adding some tests to ensure that:

1. An error is raised when `opts[:non_id_actions]` is not a list
2. We use the model name if and only if the action is in the merged list.
3. The authorization is performed correctly in these cases.